src/components/forms/home-page-slide-form/tests/home-page-slide-form.spec.js

```diff
@@ -0,0 +1,65 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { ThemeProvider } from '@material-ui/styles';
+import HomePageSlideForm from '../home-page-slide-form';
+import { theme } from '../../../app/app-theme/app.theme';
+import {
+  CREATE_SLIDE_TITLE,
+  SAVE_TITLE,
+  slideId,
+  slideOrder,
+  withId,
+  withoutId
+} from './home-page-slide-form.variables';
+
+jest.mock('react-redux');
+jest.mock('connected-react-router', () => ({
+  push: jest.fn()
+}));
+
+const themeValue = theme('light');
+const dispatch = jest.fn();
+
+useDispatch.mockImplementation(() => dispatch);
+
+describe('component tests', () => {
```

Comment:
It will be better if you add component title

Reply:
done